docs/users/step-debugging.md

```diff
@@ -91,11 +91,36 @@ An example configuration:
 ### Visual Studio Code (vscode) Debugging Setup
 
 1. Install the [php-debug](https://marketplace.visualstudio.com/items?itemName=felixfbecker.php-debug) extension.
-2. Add to the launch.json the stanza defining "Listen for xdebug" (see [config snippet](snippets/vscode_listen_for_xdebug_snippet.txt))
+2. Add `.vscode/launch.json` file with following:
```

Comment:
In keeping with the existing documentation, shouldn't this go into the vscode_listen_for_xdebug_snippet.txt?

I really don't think that version number belongs there though. I' e never needed the "launch" stuff before.